### ISSUE TITLE
Add the ability for basic automated output plots

### ISSUE THREAD

**Issue by JaminMartin** [JaminMartin/spcs_instruments#14]
Adding the ability to produce automated graphical outputs once a measurement is finished could be a nice workflow improvement. 

This could be useful for standardised data logging purposes, not publication quality figures. 

It could work something like this in the `config.toml` file.

```toml

[insights.xyscatter]
x = "device.device_name.data.some_measurement_value"
y = "device.some_other_device_name.data.some_measurement_value"

[insights.lineplot]
# ...
# ...

[insights.heatmap2d]
# ...
# ...
```

**Comment by JaminMartin** [comment]
This is currently on hold, but is enabled partially via 70d047b by importing from /tmp/pyfex.toml in your python script. This will always have an up to date in sync version of your output file for any currently running. Closing for now but will reactivate if a more integrated approach is required.
